config/prow/config.yaml

```diff
@@ -226,3 +226,43 @@ presets:
   env:
     - name: GOPPC64
       value: power9
+# knative powervs preset
+- labels:
+    preset-knative-powervs: "true"
+  env:
+    - name: IBMCLOUD_API_KEY
+      valueFrom:
+        secretKeyRef:
+          name: knative-apikey
+          key: apikey
+    - name: GITHUB_TOKEN
+      valueFrom:
+        secretKeyRef:
+          name: github-token
+          key: token
+    - name: KO_FLAGS
+      value: --platform=linux/ppc64le
+    - name: PLATFORM
+      value: linux/ppc64le
+    - name: KO_DOCKER_REPO
+      value: na.artifactory.swg-devops.com/sys-linux-power-team-ftp3distro-docker-images-docker-local/knative
```

Comment:
Try avoid using the internal registry here, we usually dont maintain any credentials which talks to IBM artifacts.

Reply:
@mkumatag  - can utilize icr.io registry in above use case , would it be fine to use that ?

Reply:
If it is only for the test then use the GitHub registry itself or quay which doesn't include any cost

Reply:
@mkumatag  we use the jfrog/icr registry for storing the ppc64le specific images which aren't multi arch & the images built during test runtime.